input/drivers/sdl_input.c

```diff
@@ -483,3 +481,17 @@ input_driver_t input_sdl = {
 #endif
    NULL
 };
+
+#if WEBOS
```

Comment:
When I think about it - I think this should've been `#ifdef` or `#if defined(...)`?

Reply:
Ah yes. Better to be `#ifdef`. Altouth usually it works as `WEBOS` was set to 1.

Reply:
Yeah, I am more wondering what'd happen if `WEBOS` wasn't defined (as per non-webos builds)